content/use/singularity.md

```diff
@@ -65,8 +65,22 @@ cat 1>&2 <<END
 
    user: ${USER}
    password: ${PASSWORD}
+
+When done using RStudio Server, terminate the job by:
+
+1. Exit the RStudio Session ("power" button in the top right corner of the RStudio window)
+2. Issue the following command on the login node:
+
+      scancel -f ${SLURM_JOB_ID}
 END
 
+# User-installed R packages go into their home directory
+if [ ! -e ${HOME}/.Renviron ]
```

Comment:
I think you want to indent this four spaces or place in a fenced code block?

Reply:
It should be part of the same fenced code block as the heredoc above it and "singularity exec" command below it, and I was hoping it could be at the same indentation level as the "singularity exec". It looks like it GitHub renders the markdown OK; does netlify have special requirements?